Add program for 2023-09-21 (#30)
rust-aarhus/meetups: meetups/2023/2023-09-21-talks/program.md

# Rust Aarhus meetup at Concordium

> 

Author: ```suggestion
Date: 2023-09-21

```

# Rust Aarhus meetup at Concordium
Date: 2023-09-21

Rust Aarhus invites you to this free meetup for all interested in the programming language [Rust].

We are still working on the program, but it will be something like the following:

- 18:05 - 18:15: Welcome (10 min)

> - 18:20 - 18:50: "Duke nukem" by Jeppe and Rasmus (30 min)

Author: ```suggestion
- 18:20 - 18:50: "Playing Duke Nukem 3D via Rust" by Jeppe and Rasmus (30 min)
```

- 18:50 - 19:00: A word from our sponsor (5-10 min)
- 19:00 - 19:45: Light dinner sponsored by Concordium
- 19:45 - 20:30: "Concordium talk" by Concordium (45 min)
- 20:35 - 20:45: "An innocent story of contributing to a Rust project" by Ole Friis Østergaard (10 min)

> - 20:50 - 21:00: Closing remarks (10 min)

Author: ```suggestion
- 20:45 - 20:55: Book lottery by Niels Skovmand (10 min)
- 20:55 - 21:00: Closing remarks (5 min)
```

- 20:55 - 21:00: Closing remarks (5 min)
- 21:00: Let's continue the discussion at the nearby bar [INSERT BAR NAME].

## Playing Duke Nukem 3D via Rust
[INFORMATION ABOUT THE SPEAKERS]

[ADD DESCRIPTION OF THE TALK HERE]

## [TITLE OF CONCORDIUM TALK]
[INFORMATION ABOUT THE SPEAKERS]

[ADD DESCRIPTION OF THE TALK HERE]

## An innocent story of contributing to a Rust project

> @olefriis works at GitHub (you're looking at it!), mainly does Ruby, but also sometimes does Rust in his spare time.

Author: ```suggestion
[Ole Friis Østergaard][ofo] works at GitHub, mainly does Ruby, but also sometimes does Rust in his spare time.
```

> We'll talk about a time when @olefriis contributed functionality to an existing Rust crate. The journey took an interesting turn at some point, and he learned things he never thought he would learn. Now you will learn it too, and you will never un-learn it.

Author: ```suggestion
We'll talk about a time when [Ole][ofo] contributed functionality to an existing Rust crate. The journey took an interesting turn at some point, and he learned things he never thought he would learn. Now you will learn it too, and you will never un-learn it.
```

## Location
This meetup will be at [INSERT ADDRESS HERE]. Please feel free to use the event chat on meetup to organize transportation if needed.

## Food
If you have special needs regarding food (allergies, vegan, etc.), please forward your request directly to philipkristoffersen@gmail.com.

> [rust]: https://www.rust-lang.org/

Author: ```suggestion
[rust]: https://www.rust-lang.org/
[ofo]: https://github.com/olefriis
```

# Rust Aarhus meetup at Concordium
Date: 2023-09-21

Rust Aarhus invites you to this free meetup for all interested in the programming language [Rust].

We are still working on the program, but it will be something like the following:

- 18:05 - 18:15: Welcome (10 min)
- 18:20 - 18:50: "Duke nukem" by Jeppe and Rasmus (30 min)
- 18:50 - 19:00: A word from our sponsor (5-10 min)
- 19:00 - 19:45: Light dinner sponsored by Concordium
- 19:45 - 20:30: "Concordium talk" by Concordium (45 min)
- 20:35 - 20:45: "An innocent story of contributing to a Rust project" by Ole Friis Østergaard (10 min)
- 20:45 - 20:55: Book lottery by Niels Skovmand (10 min)
- 20:55 - 21:00: Closing remarks (5 min)
- 21:00: Let's continue the discussion at the nearby bar [INSERT BAR NAME].

> ## [TITLE OF DUKE NUKEM TALK]

Reviewer: ```suggestion
## Playing Duke Nukem 3D via Rust
```

Reviewer: @herbstein

[INFORMATION ABOUT THE SPEAKERS]

[ADD DESCRIPTION OF THE TALK HERE]

## [TITLE OF CONCORDIUM TALK]
[INFORMATION ABOUT THE SPEAKERS]

[ADD DESCRIPTION OF THE TALK HERE]

## An innocent story of contributing to a Rust project
[Ole Friis Østergaard][ofo]: [INFORMATION ABOUT THE SPEAKERS]

> [ADD DESCRIPTION OF THE TALK HERE]

Reviewer 2: @olefriis works at GitHub (you're looking at it!), mainly does Ruby, but also sometimes does Rust in his spare time.

We'll talk about a time when @olefriis contributed functionality to an existing Rust crate. The journey took an interesting turn at some point, and he learned things he never thought he would learn. Now you will learn it too, and you will never un-learn it.

> [Ole Friis Østergaard][ofo]: [INFORMATION ABOUT THE SPEAKERS]
> 
> [ADD DESCRIPTION OF THE TALK HERE]

Reviewer 2: ```suggestion
@olefriis works at GitHub (you're looking at it!), mainly does Ruby, but also sometimes does Rust in his spare time.

We'll talk about a time when @olefriis contributed functionality to an existing Rust crate. The journey took an interesting turn at some point, and he learned things he never thought he would learn. Now you will learn it too, and you will never un-learn it.
```

## Location
This meetup will be at [INSERT ADDRESS HERE]. Please feel free to use the event chat on meetup to organize transportation if needed.

## Food

> If you have special needs regarding food (allergies, vegan, etc.), please forward your request directly to [INSERT NAME OF HOST].

Reviewer 3: ```suggestion
If you have special needs regarding food (allergies, vegan, etc.), please forward your request directly to philipkristoffersen@gmail.com.
```

I can be responsible for this, since I am vegetarian myself anyways.

[rust]: https://www.rust-lang.org/

> [ofo]: https://www.linkedin.com/in/ole-friis-%C3%B8stergaard-5827691/

Reviewer 2: ```suggestion
```

...as my GitHub profile is linked above anyway...

> This meetup will be at [INSERT ADDRESS HERE]. Please feel free to use the event chat on meetup to organize transportation if needed.

Author: ```suggestion
This meetup will be at Concordium, Åboulevarden 31, 1st floor. Please feel free to use the comments on this meetup event to organize transportation if needed.
```

- Client - server communication using [tokio_util::codec](https://docs.rs/tokio-util/latest/tokio_util/index.html)::[codec](https://docs.rs/tokio-util/latest/tokio_util/codec/index.html#)
- Server architecture for message handling
- Parsing log files using [peg](https://docs.rs/peg/latest/peg/#) parser generator

## [TITLE OF CONCORDIUM TALK]
[INFORMATION ABOUT THE SPEAKERS]

[ADD DESCRIPTION OF THE TALK HERE]

## An innocent story of contributing to a Rust project
[Ole Friis Østergaard][ofo] works at GitHub, mainly does Ruby, but also sometimes does Rust in his spare time.

We'll talk about a time when [Ole][ofo] contributed functionality to an existing Rust crate. The journey took an interesting turn at some point, and he learned things he never thought he would learn. Now you will learn it too, and you will never un-learn it.

## Location
This meetup will be at Concordium, Åboulevarden 31, 1st floor. Please feel free to use the comments on this meetup event to organize transportation if needed.

## Food
If you have special needs regarding food (allergies, vegan, etc.), please forward your request directly to philipkristoffersen@gmail.com.

[rust]: https://www.rust-lang.org/

> [ofo]: https://github.com/olefriis

Author: ```suggestion
[ofo]: https://github.com/olefriis
[jepify]: https://github.com/jepify
[herbstein]: https://github.com/Herbstein
```

# Rust Aarhus meetup at Concordium
Date: 2023-09-21

Rust Aarhus invites you to this free meetup for all interested in the programming language [Rust].

We are still working on the program, but it will be something like the following:

- 18:05 - 18:15: Welcome (10 min)
- 18:15 - 18:50: "Playing Duke Nukem 3D via Rust" by Jeppe and Rasmus (30+5 min)
- 18:50 - 19:00: A word from our sponsor (5-10 min)
- 19:00 - 19:40: Light dinner sponsored by Concordium
- 19:40 - 20:30: "Concordium talk" by Concordium (45+5 min)
- 20:35 - 20:45: "An innocent story of contributing to a Rust project" by Ole Friis Østergaard (10 min)
- 20:45 - 20:55: Book lottery by Niels Skovmand (10 min)
- 20:55 - 21:00: Closing remarks (5 min)
- 21:00: Let's continue the discussion at the nearby Die Bierbar.

## Playing Duke Nukem 3D via Rust
[Jeppe Kristensen][jepify] works at LEGO in the infrastructure department doing automation of network related tasks using Python and Typescript.

> [Rasmus Larsen][herbstein] `todo!();`

Reviewer 4: ```suggestion
[Rasmus Larsen][herbstein] works on Falcon LogScale at CrowdStrike doing provisioning and developing multi-tenancy features.
```
